### PR TITLE
Add MANIFEST.in for PyPI release #183

### PR DESCRIPTION
**Pull request by hrshdhgd**
This includes `sssom/sssom.yaml` into the distribution file thus fixing #183

### CODE REVIEW

**Comment by cthoyt** [comment]
@hrshdhgd I just added a manifest check script. please look over the new test failures and try to address them. Example manifest file (for inspiration, don't copy/paste): https://github.com/biopragmatics/bioregistry/blob/main/MANIFEST.in

**Comment by hrshdhgd** [comment]
Thanks @cthoyt , so will this mean some code in `setup.cfg` will be redundant?

**Comment by cthoyt** [comment]
No, there is zero overlap between the functionality of the MANIFEST.in file and the setup.cfg. The MANIFEST.in says what should be included in packaging and the setup.cfg adds scary non-intuitive stuff on top of it.

**Comment by hrshdhgd** [comment]
I think all I needed was `recursive-include sssom *.yaml`

But `check-manifest` recommends a lot of files that I am not sure is necessary to include with the package:

```
include *.bat
include *.md
include *.sh
include Makefile
include tox.ini
recursive-include docs *.css
recursive-include docs *.eot
recursive-include docs *.html
recursive-include docs *.inv
recursive-include docs *.js
recursive-include docs *.nojekyll
recursive-include docs *.png
recursive-include docs *.svg
recursive-include docs *.ttf
recursive-include docs *.txt
recursive-include docs *.woff
recursive-include docs *.woff2
recursive-include schema *.json
recursive-include schema *.jsonld
recursive-include schema *.py
recursive-include schema *.yaml
recursive-include sphinx *.bat
recursive-include sphinx *.py
recursive-include sphinx *.rst
recursive-include sphinx Makefile
recursive-include tests *.json
recursive-include tests *.jsonld
recursive-include tests *.md
recursive-include tests *.omn
recursive-include tests *.owl
recursive-include tests *.rdf
recursive-include tests *.sh
recursive-include tests *.tsv
recursive-include tests *.ttl
recursive-include tests *.xml
recursive-include tests *.yaml
recursive-include tests *.yml
```
Also 

```
ists of files in version control and sdist do not match!
missing from VCS:
  tests/data/basic-cliquesummary-stats.tsv
  tests/data/basic-cliquesummary.tsv
  tests/tmp/a_equivalentclass_c.sssom.tsv
  tests/tmp/a_subclassof_b.sssom.tsv
  tests/tmp/basic.tsv.json
  tests/tmp/basic.tsv.jsonld
  tests/tmp/basic.tsv.owl
  tests/tmp/basic.tsv.rdf
  tests/tmp/basic.tsv.roundtrip.json
  tests/tmp/basic.tsv.roundtrip.owl
  tests/tmp/basic.tsv.roundtrip.rdf
  tests/tmp/basic.tsv.roundtrip.tsv
  tests/tmp/basic.tsv.tsv
  tests/tmp/basic2.tsv.json
  tests/tmp/basic2.tsv.jsonld
  tests/tmp/basic2.tsv.owl
  tests/tmp/basic2.tsv.rdf
  tests/tmp/basic2.tsv.roundtrip.json
  tests/tmp/basic2.tsv.roundtrip.owl
  tests/tmp/basic2.tsv.roundtrip.rdf
  tests/tmp/basic2.tsv.roundtrip.tsv
  tests/tmp/basic2.tsv.tsv
  tests/tmp/basic_subset.json
  tests/tmp/c_equivalentclass_b.sssom.tsv
  tests/tmp/c_equivalentclass_d.sssom.tsv
  tests/tmp/clique_1.sssom.tsv
  tests/tmp/clique_10.sssom.tsv
  tests/tmp/clique_11.sssom.tsv
  tests/tmp/clique_12.sssom.tsv
  tests/tmp/clique_2.sssom.tsv
  tests/tmp/clique_3.sssom.tsv
  tests/tmp/clique_4.sssom.tsv
  tests/tmp/clique_5.sssom.tsv
  tests/tmp/clique_6.sssom.tsv
  tests/tmp/clique_7.sssom.tsv
  tests/tmp/clique_8.sssom.tsv
  tests/tmp/clique_9.sssom.tsv
  tests/tmp/cliquesummary_output_file.tsv
  tests/tmp/cliquesummary_output_url.tsv
  tests/tmp/cob-to-external.tsv.json
  tests/tmp/cob-to-external.tsv.jsonld
  tests/tmp/cob-to-external.tsv.owl
  tests/tmp/cob-to-external.tsv.rdf
  tests/tmp/cob-to-external.tsv.roundtrip.json
  tests/tmp/cob-to-external.tsv.roundtrip.owl
  tests/tmp/cob-to-external.tsv.roundtrip.rdf
  tests/tmp/cob-to-external.tsv.roundtrip.tsv
  tests/tmp/cob-to-external.tsv.tsv
  tests/tmp/converted_basic_file.json
  tests/tmp/converted_basic_file.owl
  tests/tmp/converted_basic_file.rdf
  tests/tmp/converted_basic_file.tsv
  tests/tmp/converted_basic_url.json
  tests/tmp/converted_basic_url.owl
  tests/tmp/converted_basic_url.rdf
  tests/tmp/converted_basic_url.tsv
  tests/tmp/correlation_output_file.tsv
  tests/tmp/correlation_output_url.tsv
  tests/tmp/crosstab_output_file.tsv
  tests/tmp/crosstab_output_url.tsv
  tests/tmp/d_equivalentclass_a.sssom.tsv
  tests/tmp/deduped_basic_file.tsv
  tests/tmp/deduped_basic_url.tsv
  tests/tmp/diff_output_file.tsv
  tests/tmp/diff_output_url.tsv
  tests/tmp/merged_msdf_file.tsv
  tests/tmp/merged_msdf_url.tsv
  tests/tmp/parsed_basic_file.tsv
  tests/tmp/parsed_basic_url.tsv
  tests/tmp/rewired-cob.ttl
  tests/tmp/test_parse_alignment_minidom.tsv
  tests/tmp/test_parse_obographs.tsv
  tests/tmp/test_parse_sssom_dataframe.tsv
  tests/tmp/test_parse_sssom_dataframe_url.tsv
  tests/tmp/test_parse_sssom_json.tsv
  tests/tmp/test_parse_sssom_rdf.tsv
  tests/tmp/test_parse_tsv.tsv
  tests/tmp/test_write_sssom_dataframe.tsv
  tests/tmp/test_write_sssom_json.json
  tests/tmp/test_write_sssom_owl.owl
  tests/tmp/test_write_sssom_rdf.rdf
  tests/tmp/test_write_sssom_rdf.rdf.tsv
  tests/tmp/tests_basic.json
  tests/tmp/x_equivalentclass_y.sssom.tsv
  tests/tmp/x_equivalentclass_z.sssom.tsv
  tests/tmp/y_equivalentclass_x.sssom.tsv
  tests/tmp/y_equivalentclass_z.sssom.tsv
  tests/tmp/z_equivalentclass_x.sssom.tsv
  tests/tmp/z_equivalentclass_y.sssom.tsv
```

Do I have to add all of these files?

**Comment by hrshdhgd** [comment]
@cthoyt :
`# TODO these should use the tmpfile harness` 
I did not follow. What does it mean?

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# Don't include anything in these directories
+prune docs
+prune schema
+
+# TODO these should use the tmpfile harness
+prune tests/tmp
+prune tests/data
+
+recursive-include sphinx *.py
+recursive-include sphinx *.rst
```
Comment (hrshdhgd) [inline MANIFEST.in:14]: @cthoyt :  sphinx is for documentation in GitHub-pages. Should this be packaged?

Comment (cthoyt) [reply MANIFEST.in:14]: that's a really good question. I've been doing this for a while but it's not actually used by the code, so maybe we should skip it for now

**Comment by cthoyt** [comment]
> @cthoyt : `# TODO these should use the tmpfile harness` I did not follow. What does it mean?

I'm not a big fan of the way the tests are written now - they actually create files that end up in the repository (luckily they're in .gitignore). It would be better to use `tmpfile` to create temporary files for the tests. This is just a note for later.

**Comment by hrshdhgd** [comment]
@cthoyt , do you deem this PR to be merge ready?

**Review by cthoyt — APPROVED** [review]

**Pull request merged**